tests/Unit/Helpers/Tests/Test_MakeRandomVectorInMagnitudeRange.cpp

```diff
@@ -0,0 +1,149 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <limits>
+
+#include "DataStructures/Tensor/EagerMath/DeterminantAndInverse.hpp"
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "Framework/TestHelpers.hpp"
+#include "Helpers/DataStructures/MakeRandomVectorInMagnitudeRange.hpp"
+
+template <typename DataType, size_t Dim, typename Fr = Frame::Inertial>
+tnsr::ii<DataType, Dim, Fr> random_spatial_metric(
```

Comment:
I replaced it with the helper function. I also replaced it in all other tests that reimplement `random_spatial_metric`, so that nobody will follow my example again.

I also had to adapt the helpers function, so that I can create the metric in any frame.